addon/-private/compatibility.js

```diff
@@ -0,0 +1,12 @@
+export const wait = (function() {
+  const hasLegacyEmberTestHelpers = window.require.has('ember-test-helpers');
+  const hasLatestEmberTestHelpers = window.require.has('@ember/test-helpers');
```

Comment:
The only way I could come up with to test the error case was to use the `window.require` provided by Ember. Is there any reason not to do this?

Reply:
You mean because of the stubbing, right? I think it would work to do `import require from 'require'` in this code, and then:

```javascript
import require from 'require';

test('blah blah', function(assert) {
  require.has = () => false;
  // run test
});
```

Did you try that?

Reply:
Yeah, that was my first approach. No dice.

Reply:
What went wrong? That's confusing to me because when I run a test that does `import require from 'require'`, `window.require === require` evaluates to `true`...so I'm not sure how the two are different? Unless you were doing `const { has } = require` or something in `compatibility.js`?

Reply:
Hmmm, I'm not seeing `window.require === require` evaluating to true...I believe Ember provides its own `window.require`, which is different from the imported `require`.